content/md/user-management/configure-saml-sso.md

```diff
@@ -0,0 +1,121 @@
+
+# Configure your Service Provider (SP) in your PIM to communicate with the IdP
+
+## Activate/Disable the Single Sign-On feature
+
+### Activate the Single Sign-On feature
+
+To activate the Single Sign-On feature, just click on the `SSO enabled` button in the interface.
+Once enabled, all the authentication requests will be redirected to the authentication server you specified.
+
+### Disable the Single Sign-On feature
+
+To disable the Single Sign-On feature, just click on the `SSO enabled` button in the interface.
+Once disabled, all the authentication requests will go through the classical login interface.
+
+## Configure the Identity Provider (IdP) section
+
+Using the information you gather previously, you can fill all the required fields that are required to communicate with the IdP.
+The mandatory values are the following:
+* `Entity ID`: String that uniquely identify your IdP (it is in general provided by your IdP)
+* `Sign-on URL`: URL that will be used when a user tries to log in the PIM
+* `Logout URL`: URL that will be used when a user ask for logout action in the PIM
+* `Public certificate`: As the communications are encrypted between the SP and the IdP, the IdP public key is required in order to decrypt any incoming message.
+
+:::info
+Certificates must be in X.509 format.
+:::
+
+## Configure the Service Provider (SP) section
+
+The information required for configuring the SP are the following:
+* `Entity ID`: String that uniquely identify your SP (this information needs to be provided to the IdP)
+* `Public and Private key certificates`: As the communications are encrypted, the private and public keys are required to encrypt outgoing messages.
+
+:::info
+By default, the SP configuration is automatically populated during the first access to the SSO configuration page. If you want to provide your own certificate, you can paste the public and private key certificates in the corresponding fields.
+:::
+
+:::info
+The SP certificate is self-signed and is generated to never expire. You will never have to renew it.
+:::
+
+# Tips and tricks for configuring the SAML-based SSO
+
+## About users in the PIM
+
+Once the SSO will be activated, the passwords that exists on the PIM will no longer be used.
+On the other hand, the users created in the PIM will still be needed to make the connection between the users coming from the IdP and the user rights existing in the PIM.
+
+You will have to make sure that all the potential users in your IdP that can access the PIM have an account configured in the PIM.
+Unknown users are not generated on-the-fly when accessing the PIM. If a user doesn't exist in the PIM, he will not be able to access the PIM and an error will be raised.
+
+## About user claims
+
+User claims are information that the Service Provider expects when a authentication is made on the IdP and the positive response comes back to the PIM.
+The IdP response must contain the following attributes:
```

Comment:
Is it something the integrator should make sure to make appear in the IdP response? if so, I think you can state it very explicitly.

Reply:
Yep. Reworked this part to make it clearer.